src/System.Runtime.CompilerServices.Unsafe/src/System.Runtime.CompilerServices.Unsafe.il

```diff
@@ -157,6 +157,50 @@
         ret
   } // end of method Unsafe::AsRef
 
+  .method public hidebysig static !!TTo& As<TFrom,TTo>(!!TFrom& source) cil managed aggressiveinlining
+  {
+        .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+        .maxstack 1
+        ldarg.0
+        ret
+  } // end of method Unsafe::AsRef
+
+  .method public hidebysig static !!T& Add<T>(!!T& source, int32 elementOffset) cil managed aggressiveinlining
+  {
+        .custom instance void System.Runtime.Versioning.NonVersionableAttribute::.ctor() = ( 01 00 00 00 )
+        .maxstack 3
+         ldarg.0
+         ldarg.1
+         sizeof !!T
+         conv.i
+         mul
+         add
+         ret
+  }
```

Comment:
Nit: Maybe comments be added at the end here like the other methods in this class, or the above one should be deleted